diff --git a/platform/javascript/detect.py b/platform/javascript/detect.py
index e04e9a9cab..00f8406dee 100644
--- a/platform/javascript/detect.py
+++ b/platform/javascript/detect.py
@@ -91,7 +91,8 @@ def configure(env):
     else:
         # Disable exceptions and rtti on non-tools (template) builds
         # These flags help keep the file size down.
-        env.Append(CCFLAGS=["-fno-exceptions", "-fno-rtti"])
+        # env.Append(CCFLAGS=["-fno-exceptions", "-fno-rtti"])
+        env.Append(CCFLAGS=["-fno-rtti"])
         # Don't use dynamic_cast, necessary with no-rtti.
         env.Append(CPPDEFINES=["NO_SAFE_CAST"])
 
